menyer/kackerlackan/stylesheet.css

```diff
@@ -0,0 +1,88 @@
+/*För bodyn*/
+body {
+    background-image: url(assets/background.jpg);
+    background-size: cover;
+    margin: 3% 20% 5% 20%;
+    font-family: Arial, Helvetica, sans-serif;
+    font-size: 20px;
+
+}
+
+/*Headings position*/
+h1, h2 {
+    text-align: center;
+    font-weight: 900;
+}
+
+/*för sectionerna, samt header och footer*/
+#forratter, #huvudratter, #efterratter, footer, header {
+    color: #f5f5f5;
+    padding: 10px 20px 10px 20px;
+    border-radius: 30px;
+    border-style: double hidden hidden double;
+    border-width: 10px;
+    border-color: #f5f5f5;
+    margin: 5% 0% 5% 0%;
+    background: rgba(0, 0, 0, 0.8);
+}
+
+
+/*Affect omg tag, har en overflow i item klasserna*/
+img {
+    float: left;
+    padding: 0px 30px 30px 0px;
+
+
+}
```

Comment:
Empty lines without purpose are a sin! Use one empty line per declaration. And no empty lines inside declarations.